tests/unit/lms/views/basic_lti_launch_test.py

```diff
@@ -12,6 +12,75 @@
 from lms.views.basic_lti_launch import BasicLTILaunchViews
 
 
+def canvas_file_basic_lti_launch_caller(context, pyramid_request):
```

Comment:
The "caller" naming style is from the stdlib's `methodcaller`: https://docs.python.org/3/library/operator.html#operator.methodcaller